az-digital/az_quickstart
# Closes #2529: Remove DB updates added before 2.6.0.

## Description
Uses [`hook_update_last_removed()`](https://api.drupal.org/api/drupal/core%21lib%21Drupal%21Core%21Extension%21module.api.php/function/hook_update_last_removed/8.2.x) to remove DB updates added before 2.6.0.

## Related issues




Closes #2529 

## How to test


- Verify that applying DB updates to a site that was running 2.6.0 or later works correctly.
- Verify that applying DB updates to a site that was running <2.6.0 causes Drupal to complain.

## Types of changes


### Arizona Quickstart (install profile, custom modules, custom theme)
- **Patch release changes**
- **Minor release changes**
- **Other or unknown**

### Drupal core
- **Patch release changes**
- **Minor release changes**
- **Other or unknown**

### Drupal contrib projects
- **Patch release changes**
- **Minor release changes**
- **Other or unknown**

## Checklist
